client/src/components/Nav.jsx

```diff
@@ -67,7 +68,7 @@ class Navbar extends React.Component {
     return (
       <div>
         <AppBar
-          title="TrekTracker"
+          title={<TrekTracker/>}
```

Comment:
Being curious. I have never seen passing down entire component variable as props.

Is this intended?

Reply:
@jhjaylim @nickelow Yea, it is intended. The material-ui nav component doesn't currently allow adding a image logo next to the title. I couldn't find a fitting glyphicon and don't think it is important enough to take the time to try to add an SVG, so I implemented a relevant version of this solution, https://github.com/callemall/material-ui/issues/5478

Reply:
[
<img width="345" alt="screen shot 2017-08-27 at 9 16 46 pm" src="https://user-images.githubusercontent.com/17533722/29759105-a449d618-8b6d-11e7-91ec-be54d5ddb34c.png">
](url) this is what it looks like. Totally cosmetic.

Reply:
good! merging!